app/views/works/_mystery_blurb.html.erb

```diff
@@ -1,15 +1,11 @@
 <div class="mystery header picture module">
   <h4 class="heading"><%= item.is_a?(Work) ? ts("Mystery Work") : ts("Mystery Series") %></h4>
-  <% if item.is_a?(Work) %>
-    <h5 class="heading"><%= h(ts("Part of ")) + (unrevealed ? item.collections.unrevealed : item.collections.anonymous).map {|challenge| link_to(challenge.title, collection_path(challenge))}.join(", ").html_safe %></h5>
+  <% if item.is_a?(Work) && !item.approved_collections.unrevealed.empty? %>
```

Comment:
We could probably get away with just `<% if item.approved_collections.unrevealed.present? %>` or `<% unless item.approved_collections.unrevealed.empty? %>` here: despite what line two says, I don't see any indication this blurb is actually ever used for series.